### ISSUE TITLE
Accessibility: Ensure aria-hidden elements are not focusable nor contain focusable elements

### ISSUE THREAD

**Issue by bedmison** [web-cat/web-cat-subsystem-Core#13]
Ensure aria-hidden elements are not focusable nor contain focusable elements. [See more information](https://dequeuniversity.com/rules/axe/4.10/aria-hidden-focus?application=AxeChrome)

This occurs in the Detailed Reference Test Results pane, on the show/hide disclosure arrow for the test case file name. 

To solve this problem, you need to fix the following:

- Focusable content should have `tabindex="-1"` or be removed from the DOM
